owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewAudioFragment.kt

```diff
@@ -342,6 +359,15 @@ class PreviewAudioFragment : FileFragment() {
         }
     }
 
+    private fun showSnackbarMessage(){
+        val snackbarString = if (file.isAvailableOffline){
+            R.string.confirmation_unset_available_offline
+        } else {
+            R.string.confirmation_set_available_offline
+        }
+        Snackbar.make(requireView(), snackbarString, Snackbar.LENGTH_LONG).show()
+    }
+
```

Comment:
I don't know if it is worth it to separate this in another method when you can do the same in-place with just 1 line and you only call it twice, one per each `if` branch. Moreover, the condition would be the opposite I think: if `file` is available offline is because you just set it (so, you just pressed the `Set as available offline` option and thus the message in the Snackbar should say that, not the opposite)

Reply:
I've removed the method and now the `Snackbar` string depends on the condition.